src/Microsoft.DotNet.Build.Tasks/GenerateResourcesCode.cs

```diff
@@ -169,6 +174,18 @@ private void WriteDebugCode()
             }
         }
 
+        private void WriteGetTypeProperty()
+        {
+            if (_targetLanguage == TargetLanguage.CSharp)
+            {
+                _targetStream.WriteLine("        internal static Type ResourceType {1}{3}              get {1} return typeof({0}); {2}{3}        {2}", _resourcesName + ".SR", "{", "}", Environment.NewLine);
```

Comment:
This property is used in the SR.cs file checked under common which is creating ResourceManager instance. you'll see this in other code review I sent to you yesterday. 

thanks for your review. I'll merge this change after you sign off on the other code review just in case we need to adjust anything else.